src/_includes/navbar.html

```diff
@@ -1,12 +1,9 @@
 <header class="navbar">
   <ul aria-label="Main navigation">
```

Comment:
Should we remove this aria-label? As I'm not sure this is functioning as a navigation? There is a label on the side/expandable nav